src/components/SidebarLink.vue

```diff
@@ -20,7 +20,7 @@ export default {
 </script>
 
 <template>
-    <router-link :to="to ? to : '/signIn'" class="link" :class="{active: isActive}">
+    <router-link :to="to ? to : ''" class="link" :class="{active: isActive}">
```

Comment:
So this was here to redirect the user back to the signin page when they logged out. Without it, they stay on the page they were on even through they are signed out

Reply:
Ah see I was testing how to make it not move you if the link was empty because I was trying to get the currency exchange to be more of a pop up rather than its own page. I abandoned that so we can change this back to your implementation